src/main/java/teammates/ui/pagedata/PageData.java

```diff
@@ -850,4 +856,101 @@ public void setStatusMessagesToUser(List<StatusMessage> statusMessagesToUser) {
         return statusMessagesToUser;
     }
 
+    /**
+     * Builds template that will be used by respondents to add comments to their responses.
+     * @param question question of response
+     * @param responseId id of response
+     * @param giverName name of person/team giving comment
+     * @param recipientName name of person/team receiving comment
+     * @param timezone Time zone
+     * @param isCommentFromFeedbackParticipant true if comment giver is feedback participant
+     * @return Feedback response comment add form template
+     */
+    public FeedbackResponseCommentRow buildFeedbackResponseCommentAddFormTemplate(FeedbackQuestionAttributes question,
```

Comment:
Will this function shared by other class?

Reply:
Yes, it is shared by `InstructorFeedbackResultsPageData` and `FeedbackSubmissionEditPageData`